manage.go

```diff
@@ -0,0 +1,84 @@
+//go:build manage
+// +build manage
+
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+
+	"github.com/DiscordGophers/dgobot/commands"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/peterbourgon/ff/v3"
+	"github.com/peterbourgon/ff/v3/ffcli"
+)
+
+var (
+	session *discordgo.Session
+	GuildID string
+)
+
+func main() {
+	fs := flag.NewFlagSet("dgbot", flag.ExitOnError)
```

Comment:
```suggestion
	fs := flag.NewFlagSet("dgobot", flag.ExitOnError)
```
To be consistent with the flag set name in `main.go`.